### ISSUE TITLE
Activate account-for-pod-resources again in dcos packages

### ISSUE THREAD

**Issue by sttts** [mesosphere/kubernetes-mesos#491]
Follow-up of #469.

**Comment by jdef** [comment]
we decided to wait for k8s 1.1 release before creating a release branch to house this change

**Comment by jdef** [comment]
related: https://github.com/kubernetes/kubernetes/commit/092dddd12ca4cc5e66597de66a81fc77fdd3f29b

**Comment by sttts** [comment]
From that commit:
```
+// disableKernelMemoryOvercommitHandling tells the kernel to perform no memory over-commit handling.
+// Under this setting, the potential for memory overload is increased, but so is performance for
+// memory-intensive tasks
+// sets /proc/sys/vm/overcommit_memory to 1
```

This is done from the container manager in the kubelet.

**Comment by jdef** [comment]
implemented as part of #408

**Comment by sttts** [comment]
Will be released as part of https://github.com/mesosphere/kubernetes-mesos/issues/600
